#joint names
joint_names: [left_arm_1_joint, left_arm_2_joint, left_arm_3_joint, left_arm_4_joint, left_arm_5_joint, left_arm_6_joint, left_arm_7_joint]

arm_select: left

#velocity commands topic to publish
vel_commands_topic: /left_arm_controller/command_vel

# velocity screw max limits, [m/s] [deg/s]
v_max: 0.15
w_max: 45

# gains of the contact point estimator
gamma_r: 6.0
kappa_r: 0.001
beta_r: 0.2

# initial contact point (in FT sensor frame)
initial_r: [-0.1, 0.0, 0.3]

# gains of surface normal estimator
gamma_n: 0.0 #0.001
beta_n: 0.0 #0.0001

# initial surface normal (in base frame)
initial_n: [0.0, 0.0, 1.0]


# normal force reference
normal_force_ref: 2.0

# PI gains of force feedback
alpha_f: 0.0
beta_f: 0.0

# control frequency
control_frequency: 130

# position gain of the trajectory generator
alpha_v_d: 0.0

# radius of the circle trajectory [m]
circle_traj_radius: 0.0

# period of the circle trajectory [s]
circle_traj_period: 20.0


# reference frame ids
robot_base_frame_id: /arm_base_link
robot_ft_frame_id: /left_arm_FT_sensor

#timeout in seconds
timeout: 100

